plugins/header_rewrite/expander.cc

```diff
@@ -113,6 +113,13 @@ VariableExpander::expand(const Resources &res)
         }
         TSHandleMLocRelease(bufp, TS_NULL_MLOC, url_loc);
       }
+    } else if (variable == "%<cque>") {
+      // The client request effective URL.
+      int url_len = 0;
+      const char *url = TSHttpTxnEffectiveUrlStringGet(res.txnp, &url_len);
```

Comment:
This string is allocated and must be released. Bad API design, yes, but there it is.